File: GateOne/DayThree/java/RandomQuestions.java
import java.util.Scanner;
import java.util.Random;

public class RandomQuestions{

public static void main(String[] args){

Scanner input = new Scanner(System.in);

Random rand = new Random();

int correctAnswer = 0;
int wrongAnswer = 0;

int numberOne = 0;
int numberTwo = 0;


	for(int counter = 1; counter < 11; counter++){

	int numbersOne = rand.nextInt(10);

	int numbersTwo = rand.nextInt(5);

	int answer = numbersOne + numbersTwo;

	System.out.print("what is " +numbersOne+ " + "+numbersTwo+ " = ");
	int questionAnswer = input.nextInt();

	if(questionAnswer == answer){

	correctAnswer++;

	}

	else if(questionAnswer != answer){
	
	wrongAnswer++;

	System.out.println("you got "+ correctAnswer+ " correct");
	System.out.println("you got "+ wrongAnswer+ " wrong"); 

	}


	
	}



}


}

File: GateOne/DayThree/python/RandomQuestions.py
import random 

correctAnswers = 0
wrongAnswer = 0

randomNumber = 0
randomNumberTwo = 0

for counter in range (1, 11):
	randomNumber = random_randint(1, 11)
	randomNumberTwo = random_randint(1,10)
	answer = randomNumber + randomNumberTwo

	randomQuestion = int(input(f' what is {randomNumber} + {randomNumberTwo}? '))

	if randomQuestion == answer:
		correctAnswer

	else  randomQuestion = answer:
		wrongAnswer